vyxal/structure.py

```diff
@@ -190,42 +193,42 @@ def transpile(self) -> str:
 
 
 class ListLiteral(Structure):
-    def __init__(self, branches: list[list["Structure"]]):
+    def __init__(self, branches: List[List["Structure"]]):
         super().__init__(branches)
         self.items = branches
 
     def transpile(self) -> str:
         # We have to manually build this because we don't know how
-        # many list items there will be.
```

Comment:
oopsie doopsie, somebody did a blanket find-and-replace... :þ

please fix this (just completely undo this replace as mentioned above)